subsys/bluetooth/audio/shell/bap.c

```diff
@@ -168,7 +168,7 @@ static uint16_t get_next_seq_num(uint32_t interval_us)
 
 #if defined(CONFIG_LIBLC3)
 NET_BUF_POOL_FIXED_DEFINE(sine_tx_pool, CONFIG_BT_ISO_TX_BUF_COUNT,
-			  CONFIG_BT_ISO_TX_MTU + BT_ISO_CHAN_SEND_RESERVE,
+			  BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU),
 			  8, NULL);
```

Comment:
```suggestion
			  BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU),
			  CONFIG_BT_CONN_TX_USER_DATA_SIZE, NULL);
```

Reply:
done, can you take another look?

Reply:
Did you push? i till see use of `8`...

Reply:
dang I did all of the others and this one slipped. Should be changed now